SR2024/2023-09-23-place-confirmation.md

```diff
@@ -6,32 +6,10 @@ subject: Confirming your place at Student Robotics 2024
 You're in! We're happy to confirm you've got a place at Student Robotics 2024!
 
 As the new school year starts, so does Student Robotics. The year starts with a
-"[Kickstart][kickstart]" event which gives competitors and team supervisors an
+virtual "[Kickstart][kickstart]" event which gives competitors and team supervisors an
 introduction to this year's competition and overview of how to build and
 engineer their robot.
 
-At various points in the year we will have a number of "Tech Days" at locations
-around the UK. These provide an opportunity for your team to get hands-on help
-from our volunteer mentors, make progress on their robot and meet other teams.
-
-Throughout the year your team will be able to get support for our mentors via
-[Discord][discord]. In addition to shared channels where any team can ask
-questions, each team will have their own private text chat channel and team
-supervisors can request voice & video channels too. These offer a space for
-teams to discuss their robot and get direct support from our mentors at any
-time. We will send out invites to Discord separately.
-
-This year **Kickstart is fully virtual**. We strongly encourage teams to meet up
-to watch the presentations, get to know each other and start working together on
-the kit. We will be shipping kits out to teams ahead of the event, so
-_please ensure we have the correct shipping address_.
-
-On the day we will be livestreaming an introduction to the competition, the game
-and the kits. We will also be providing a set of tasks which will familiarise
-competitors with the kit, which the teams can complete afterwards. There will be
-mentors around in Discord throughout the day to support teams working through
-these tasks.
-
 To receive your kit we need a few things from you:
```

Comment:
This is a bit clearer, removing the deadline sentence further down.
```suggestion
So that we can ship you your robotics kit, for you to use during Kickstart and the competition, we need a few things from you. In order for the kit to arrive by Kickstart please complete the following by 1st October:
```

Reply:
Thanks, I've built on this in https://github.com/srobo/team-emails/pull/153/commits/06eb11500401958bef479d8b4fc8fd468359e5c0